pyproject.toml

```diff
@@ -48,7 +48,7 @@ Home = "https://github.com/conda/conda-index"
 [tool.pytest.ini_options]
 minversion = "7.0"
 # remove --cov when testing in the debugger
-addopts = "--cov"
+# addopts = "--cov"
```

Comment:
?

Reply:
I don't like having this in the default anymore because it interferes with running the tests in a debugger.

Reply:
Right, you mentioned that before, this will disable coverage collection in CI though right?

Reply:
Moved into tests.yml